scripts/run_docker_build.sh

```diff
@@ -20,16 +20,17 @@ CONDARC
 cat << EOF | docker run -i \
                         -v ${REPO_ROOT}:/staged-recipes \
```

Comment:
Maybe we really shouldn't even mount this in `/` now that I'm thinking about it. Any thoughts on alternatives?